typings/index.d.ts

```diff
@@ -1156,10 +1234,18 @@ declare module 'discord.js' {
       filter: CollectorFilter<[MessageReaction, User]>,
       options?: AwaitReactionsOptions,
     ): Promise<Collection<Snowflake, MessageReaction>>;
+    public awaitComponentInteractions(
+      filter: CollectorFilter<[ComponentInteraction]>,
+      options?: AwaitComponentInteractionsOptions,
+    ): Promise<Collection<Snowflake, MessageReaction>>;
```

Comment:
shouldn't this return `Promise<Collection<Snowflake, ComponentInteraction>>`?

Reply:
Yep, that's my copy-pasting mistake :D